atlasdb-workload-server-distribution/src/main/java/com/palantir/atlasdb/workload/WorkloadServerLauncher.java

```diff
@@ -157,25 +161,47 @@ private void runWorkflows(WorkloadServerConfiguration configuration, Environment
 
         waitForTransactionStoreFactoryToBeInitialized(transactionStoreFactory);
 
+        List<WorkflowAndInvariants<Workflow>> workflowsAndInvariants = new ArrayList<>(List.of(
```

Comment:
As this method is getting quite big, I'd probably extract some of this code to two helper methods: `getAllWorkflowsAndInvariants` and `selectWorksflowAndInvariantsToRun` or something along those lines

Reply:
yeah, seems reasonable